server/app/models/Account.java

```diff
@@ -50,6 +52,9 @@ public class Account extends BaseModel {
   private String authorityId;
   private String emailAddress;
 
+  @DbJsonB(name = "id_tokens")
+  private SerializedIdTokens serializedIdTokens;
```

Comment:
**suggestion (blocking):** Please add a test case that exercises saving and loading id tokens to https://github.com/civiform/civiform/blob/main/server/test/models/AccountTest.java#L12

Reply:
Done